crates/astria-cli/src/commands/bridge/submit.rs

```diff
@@ -129,14 +129,17 @@ async fn submit_transaction(
         .await
         .wrap_err("failed to get nonce")?;
 
-    let tx = UnsignedTransaction {
-        params: TransactionParams::builder()
-            .nonce(nonce_res.nonce)
-            .chain_id(chain_id)
-            .build(),
-        actions,
-    }
-    .into_signed(signing_key);
+    let tx = UnsignedTransaction::builder()
+        .actions(actions)
+        .params(
+            TransactionParams::builder()
+                .nonce(nonce_res.nonce)
+                .chain_id(chain_id)
+                .build(),
+        )
+        .build()
+        .expect("failed to build transaction from actions")
```

Comment:
You are in an environment that exits early on errors. I would return an error here to potentially get some error feedbackk.